vk/backend/backend.go

```diff
@@ -146,6 +149,27 @@ func NewBackend(ctx context.Context, rp runtimeTypes.ContainerRuntimeProvider, p
 		return nil, errors.New("error converting network resource units")
 	}
 
+	kernelVersionRaw, err := os.ReadFile("/proc/sys/kernel/osrelease")
+	if err != nil {
+		kernelVersionRaw = []byte(fmt.Sprintf("problem reading kernel version: %v", err))
+	}
+
+	kernelVersion := strings.TrimSpace(string(kernelVersionRaw))
+
+	// this is actually kind of racy... if mutator runs and upgrades the
+	// package during the window between when this binary starts and this
+	// command runs, we'll render the wrong version. we could plumb through
+	// the version via golang's -ldflags -X, but we'll still have this
+	// problem if we start querying other package versions. best to do this
+	// as early as possible, I think.
+	executorVersionRaw, err := exec.Command("dpkg-query", "--show", "--showformat=${Version}", "titus-executor").CombinedOutput()
+	if err != nil {
+		executorVersionRaw = []byte(fmt.Sprintf("problem reading executor version: %v", err))
+	}
+	executorVersion := strings.TrimSpace(string(executorVersionRaw))
+
+	pod.Annotations[podCommon.AnnotationKeyRuntimeVersions] = fmt.Sprintf("kernel-version=%s,titus-executor=%s", kernelVersion, executorVersion)
```

Comment:
I think this is kinda an awkward place to put this.

Instead can you extend the *runtimeTypes.Details struct and put this csv in there? Say `VersionDetailsCSV` ?

And then find the line `update.Details.NetworkConfiguration.ToAnnotationMap()` and add an analogous line to add in the annotation there in the update